content/docs/5.1/deploy/operate/scale.md

```diff
@@ -146,12 +146,12 @@ You can take similar steps to remove the PD node.
 
     The `--node` parameter is the ID of the node to be taken offline.
 
-    If you see the `Scaled cluster <cluster-name> in successfully`, the scale-in operation is successfully completed.
-
+    If you see the message "Scaled cluster in successfully", it means that the scale-in operation is successfully completed.
+    
+    Besides, since the scale-in process takes some time, the status of the node to be taken offline became `Tombstone` also indicates that the scale-in operation is successfully completed.
```

Comment:
```suggestion
    Besides, if the status of the node to be taken offline becomes `Tombstone`, it also indicates that the scale-in operation is successfully completed because the scale-in process takes some time.
```

Reply:
rest LGTM~